indexer-app/src/cmr/indexer/data/concepts/subscription.clj

```diff
@@ -0,0 +1,44 @@
+(ns cmr.indexer.data.concepts.subscription
+  "Contains functions to parse and convert subscription concepts."
+  (:require
+   [clojure.string :as string]
+   [cmr.common.mime-types :as mt]
+   [cmr.indexer.data.elasticsearch :as es]))
+
+(defmethod es/parsed-concept->elastic-doc :subscription
+  [context concept parsed-concept]
+  (let [{:keys [concept-id revision-id deleted provider-id native-id user-id
+                revision-date format extra-fields]} concept
+        {:keys [subscription-name subscriber-id collection-concept-id]} extra-fields]
+    (if deleted
```

Comment:
nit: These two maps appear the same except for `:metadata-format`. Define one map and and do a 
```
(if-not deleted 
   (merge sub-map 
                {:metadata-format (name (mt/format-key format))}) 
   sub-map)
```

Reply:
Actually I don't need to have so many fields when deleted. Will remove a few.